documentation/operators/src/testing/node-api-check.md

```diff
@@ -12,26 +12,32 @@ Besides that, Gateway operators can check out their node performance, connectivi
 ### Basic API usage
 
 For information about available endpoints and their status, you can refer to:
-```
+```sh
+# sustitude <NODE_IP_ADDRESS> or <NODE_DOMAIN> with a real one
 # for http
-http://<IP>:8080/api/v1/swagger/#/
+http://<NODE_IP_ADDRESS>:8080/api/v1/swagger/#/
+# or
+http://<NODE_IP_ADDRESS>/api/v1/swagger/#/
 
-# for https reversed proxy
-https://<DOMAIN>/api/v1/swagger/#/
+# for reversed proxy/WSS
+https://<NODE_DOMAIN>/api/v1/swagger/#/
 ```
 
 For example to determine which mode your node is running, you can check `:8080/api/v1/roles` endpoint:
-```
+```sh
+# sustitude <NODE_IP_ADDRESS> or <NODE_DOMAIN> with a real one
 # for http
-http://<IP_ADDRESS>:8080/api/v1/roles
+http://<NODE_IP_ADDRESS>:8080/api/v1/roles
+# or
+http://<NODE_IP_ADDRESS>/api/v1/roles
 
-# for https reversed proxy
-https://<DOMAIN>/api/v1/roles
+# for reversed proxy/WSS
+https://<NODE_DOMAIN>/api/v1/roles
 ```
 
 ## `node_api_check.py`
 
-To make this a bit easier, we made a CLI tool quering all vailable API endpoints based on node `Identity Key` (further denoted as `<ID_KEY>`) called `node_api_check.py`. To diagnose your node performance, whether by yourself or by sharing an output in our [operator channel](https://matrix.to/#/#operators:nymtech.chat), this tool provides you with a quick overview of live data. We recommend to run this checker alongside [`nym_gateway_probe`](gateway-probe.md) to triage both performance and an actual routing.
+To make this a bit easier, we made a CLI tool querying all viable API endpoints based on node `Identity Key` (further denoted as `<ID_KEY>`) called `node_api_check.py`. To diagnose your node performance, whether by yourself or by sharing an output in our [operator channel](https://matrix.to/#/#operators:nymtech.chat), this tool provides you with a quick overview of live data. We recommend to run this checker alongside [`nym_gateway_probe`](gateway-probe.md) to triage both performance and an actual routing.
```

Comment:
Did you mean ' viable ' or ' available ' ?

Reply:
- [x] done